pkg/cmd/server/api/types.go

```diff
@@ -8,8 +8,8 @@ import (
 
 const (
 	FeatureBuilder    = `Builder`
-	FeatureS2I        = `S2I Builder`
-	FeatureWebConsole = `Web Console`
+	FeatureS2I        = `S2IBuilder`
```

Comment:
> ok, then do the minimal impl needed to alias these two flags, since we won't add any new flags with spaces

Yeah, that's what I aimed for. More minimalistic solution would be to move and hide deprecated names to `pkg/cmd/server/api/helpers.go` and use `FeatureList` in validation code so that they don't need to be exported. But I'd rather have them visible and close to their canonical names.